code/menuui/playermenu.cpp

```diff
@@ -946,7 +946,7 @@ int player_select_get_last_pilot()
 	return 0;
 }
 
-void player_select_init_player_stuff(int mode)
+void player_select_init_player_stuff(int mode, SCP_string const current_callsign)
```

Comment:
Why modify the function call? It seems like a value which could easily be determined in the function itself for a cleaner implementation.

Reply:
Well, this operation should only be done during half of the calls to this function.  I can make a global bool instead?